gaptrain/calculators.py

```diff
@@ -100,16 +100,10 @@ def run_autode(configuration, max_force=None, method=None, n_cores=1, kwds=None)
 
     try:
         configuration.forces = -ha_to_ev * calc.get_gradients()
-    except CouldNotGetProperty:
-        logger.error('Failed to set forces')
+        configuration.energy = ha_to_ev * calc.get_energy()
 
-    energy = calc.get_energy()
-    if energy is None:
-        logger.error('Failed to calculate the energy')
-        return configuration
-
-    configuration.energy = ha_to_ev * energy
-    configuration.partial_charges = calc.get_atomic_charges()
+    except (CalculationException, TypeError):
+        logger.error('Failed to set forces or energy')
```

Comment:
Any indication to the user which it failed to set?

Reply:
It's terminal either way so meh, I don't think it matters